backend/src/main/resources/graphql/main.graphqls

```diff
@@ -301,9 +301,11 @@ type PendingOrganization {
 }
 
 type UploadResult {
+  internalId: ID!
```

Comment:
Is this type being used anywhere on the frontend yet? Adding these new required elements might cause the frontend app to crash if so.

Reply:
Its not possible to create a db object without an internal-id since its the primary key and its required in the schema. As far as using it in the frontend, I am only using it as the `key` for react jsx map, but I am open to use the `report Id `for that and drop the internal id.

Reply:
it will be used in a follow up PR for the UI

Reply:
Ahhh ok, I thought this was an input graphQL type, but since it's an output type the new requirements shouldn't matter.